adapters/openrtb_util.go

```diff
@@ -52,7 +53,12 @@ func makeVideo(unit pbs.PBSAdUnit) *openrtb.Video {
 	mimes := make([]string, len(unit.Video.Mimes))
 	copy(mimes, unit.Video.Mimes)
 	pbm := make([]int8, 1)
+	//this will become int8 soon, so we only care about the first index in the array
 	pbm[0] = unit.Video.PlaybackMethod
+	if pbm[0] == 0 {
+		// slice of [0] = 0 will be considered non empty and sent, even though 0 is invalid per the spec.
+		pbm = append(pbm[:0], pbm[1:]...)
```

Comment:
Probably could be `pbm = nil`. It's worth testing... but most (if not all) of the methods in Go treat `nil` like an empty slice. For example: https://tour.golang.org/moretypes/12